ci/Dockerfile.lambda

```diff
@@ -17,6 +17,7 @@ WORKDIR ${TASK_ROOT}
 COPY requirements.txt ${TASK_ROOT}
 RUN set -ex && pip3 install -r requirements.txt
 RUN pip3 install awslambdaric newrelic-lambda
+RUN pip install --no-cache-dir newrelic
```

Comment:
hrm. not sure why I added this. Was it needed or was I just poking things randomly to see if I could get New Relic working? Before merge should see if things work without this line.

Reply:
ahh, we added it here when we were trying random things to get New Relic working again 😅 . Ok, let me see if we still need it.

Reply:
looks like the app works without it and still sends to New Relic 👍